### DISCUSSION TITLE
Welcome to raster_tracer Discussions!

### DISCUSSION THREAD

**Discussion by mkondratyev85** [mkondratyev85/raster_tracer#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mkondratyev85** [comment]
Hello. Test message.

**Comment by mkondratyev85** [comment]
Hello! I'm replying to email rather than answering on the site since I
couldn't find your message on github.

It looks similar to the issues some people have with projection. See this
for example:
https://github.com/mkondratyev85/raster_tracer/issues/12#issuecomment-633977240

Try to reproject your raster map. And, please raster then asking on
discussion, try to create an issue. It will be easier for others with
similar problems.
Right now I'm not even sure that you'll see my answer at all.

Mikhail.

**Comment by Patribu** [comment]
Hi,

I deleted the comment 2 minutes after creating it, as I understood that the
reference systems must all be the same to use your plugin. It's now
wording, thanks.


Best regards
Patrick

*Dr. Patrick Ribeiro* | CEO | openforests.com | FB
<https://www.facebook.com/openforests>
Portugal: +351 96 454 7777 | Germany: +49 (0)15 777 555 667 | Skype:
patrick.m.r.ribeiro

<https://explorer.land>

*Visit OpenForests' new project presentation platform at
https://explorer.land <https://explorer.land>*

OpenForests  UG (haftungsbeschränkt), Moerser Straße 171, 47803 Krefeld,
Germany, Geschäftsführer: Alexander Watson, Stefan Haas, Dr. Patrick
Ribeiro, Registergericht: HRB 15728 Amtsgericht Krefeld, USt-IdNr.:
DE279880113

**Comment by BarryCotton** [comment]
I am a geologist making maps and have to do a lot of digitizing of my interpretations. I have tried to use your Raster Tracer; followed all the instructions I can get from youtube, etc including taking care with projections.  When I start to trace from my raster I get perfect traces that are always shifted to the left by the same amount, except for the points I trace with, which are in their correct location.  Please see the attachment for an example.  I have tried several dfferent rasters including some in different countries, etc but always the same problem.
![raster tracer](https://github.com/mkondratyev85/raster_tracer/assets/129709102/8f396ba1-94f1-4ebd-a020-7669dd68365a)

**Reply by mkondratyev85** [reply]
Hello @BarryCotton, thank you for using the raster_tracer. From your screenshot it's not clear what CRS you're using for your map. It would help if you send me the map you're trying to draw on my email (visible in my profile) and I will try to replicate the issue.

**Comment by BarryCotton** [comment]
Thanks for replying and for your wonderful plugin.  I have attached the
raster which was georeferenced using Mapinfo and the .shp file vector
file.  Maybe that's the problem?  Projection is EPSG 28350 = GDA94
Mapgrid of Australia Zone 50...in Western Australia.
If I can get the Raster Tracer working I will create my interpretation
overlays differently using coloured pencils for annotation, etc.

On Sat, 16 Sept 2023 at 23:35, Mikhail Kondratyev ***@***.***>
wrote:

> Hello @BarryCotton <https://github.com/BarryCotton>, thank you for using
> the raster_tracer. From your screenshot it's not clear what CRS you're
> using for your map. It would help if you send me the map you're trying to
> draw on my email (visible in my profile) and I will try to replicate the
> issue.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mkondratyev85/raster_tracer/discussions/27#discussioncomment-7022294>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A65TILSBLPA7PT4TX7SIAM3X2XBK3ANCNFSM4U3PJ4UQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
*Barry Cotton*
*Australia: +61 (0)4 3232 5 914*
*Messenger: Barry Cotton*
*email:* ***@***.***

**Comment by BarryCotton** [comment]
Mikhail, I got it sorted.  I went back and Georeferenced the image using
the QGIS Georeferencer and it worked perfectly.  It obviously doesn't like
the Mapinfo registration.  What a wonderful plugin.  Usually when I make a
map, say for two weeks, it takes another week to digitize it, now one day
or less.
Thanks

On Sun, 17 Sept 2023 at 10:34, Barry Cotton ***@***.***> wrote:

> Thanks for replying and for your wonderful plugin.  I have attached the
> raster which was georeferenced using Mapinfo and the .shp file vector
> file.  Maybe that's the problem?  Projection is EPSG 28350 = GDA94
> Mapgrid of Australia Zone 50...in Western Australia.
> If I can get the Raster Tracer working I will create my interpretation
> overlays differently using coloured pencils for annotation, etc.
>
> On Sat, 16 Sept 2023 at 23:35, Mikhail Kondratyev <
> ***@***.***> wrote:
>
>> Hello @BarryCotton <https://github.com/BarryCotton>, thank you for using
>> the raster_tracer. From your screenshot it's not clear what CRS you're
>> using for your map. It would help if you send me the map you're trying to
>> draw on my email (visible in my profile) and I will try to replicate the
>> issue.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/mkondratyev85/raster_tracer/discussions/27#discussioncomment-7022294>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A65TILSBLPA7PT4TX7SIAM3X2XBK3ANCNFSM4U3PJ4UQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***
>> com>
>>
>
>
> --
> *Barry Cotton*
> *Australia: +61 (0)4 3232 5 914*
> *Messenger: Barry Cotton*
> *email:* ***@***.***
>
>


-- 
*Barry Cotton*
*Australia: +61 (0)4 3232 5 914*
*Messenger: Barry Cotton*
*email:* ***@***.***

**Reply by mkondratyev85** [reply]
Hello, Barry! Thank you for the wonderful review. I'm glad the plugin has been helpful to you.
